browser/src/Editor/NeovimEditor/NeovimEditor.tsx

```diff
@@ -245,15 +245,6 @@ export class NeovimEditor extends Editor implements IEditor {
             this._symbols,
         )
 
-        const updateViewport = () => {
```

Comment:
Nice, thanks for cleaning this up! Doesn't make sense anymore, now that the editor is more self-contained.